src/operator/contrib/dgl_graph.cc

```diff
@@ -464,79 +449,87 @@ static void NegateSet(const std::vector<size_t> &idxs,
 /*
  * Uniform sample
  */
-static void GetUniformSample(const std::vector<dgl_id_t>& ver_list,
-                             const std::vector<dgl_id_t>& edge_list,
+static void GetUniformSample(const dgl_id_t* val_list,
+                             const dgl_id_t* col_list,
+                             const dgl_id_t* indptr,
+                             const dgl_id_t dst_id,
```

Comment:
I think you can pass val_list + offset, col_list + offset, and size.
it'll make the arguments easier to understand and the implementation a littel simpler.